src/modules/scorelist.js

```diff
@@ -1,11 +1,31 @@
-const sampleData = [
-  { name: 'Name', score: 100 },
-  { name: 'Name', score: 20 },
-  { name: 'Name', score: 50 },
-  { name: 'Name', score: 78 },
-  { name: 'Name', score: 125 },
-  { name: 'Name', score: 77 },
-  { name: 'Name', score: 42 },
-];
-
-export default sampleData;
+import { getScores } from './api.js';
+import sortScoresDescending from './sortscores.js';
+
+// Function to display the leaderboard scores in the score list section
+const displayScores = async () => {
+  const scoreList = document.getElementById('score-list');
+
+  // Clear existing content in the score list
+  scoreList.innerHTML = '';
+
+  try {
+    const scores = await getScores();
+    const sortedScores = sortScoresDescending(scores); // Sort scores
+
+    sortedScores.forEach((entry, index) => {
+      const listItem = document.createElement('li');
+      listItem.textContent = `${entry.user}: ${entry.score}`;
+
+      // Add the 'alternate' CSS class to every second list item (even-indexed)
+      if (index % 2 !== 0) {
+        listItem.classList.add('alternate');
+      }
+
+      scoreList.appendChild(listItem);
+    });
```

Comment:
- [ ] I couldn't confirm the functionality of the `refresh button` because data were posted directly to the `UI` when I clicked `Add score`. Kindly ensure that the `Add score` button post data to the `api` and not the `UI`.

<img width="755" alt="Untitled" src="https://github.com/mryeminthu/Leaderboard/assets/103386183/87f268b9-a33e-4d0e-8317-05e2200895a3">